### DISCUSSION TITLE
articles/youtube-api%EB%A1%9C-next.js-%EB%B8%94%EB%A1%9C%EA%B7%B8%EC%97%90-bgm-%EB%A7%8C%EB%93%A4%EA%B8%B0

### DISCUSSION THREAD

**Discussion by giscus** [obvoso/obvoso-blog#33] [Comments]
# articles/youtube-api%EB%A1%9C-next.js-%EB%B8%94%EB%A1%9C%EA%B7%B8%EC%97%90-bgm-%EB%A7%8C%EB%93%A4%EA%B8%B0

react-youtube 라이브러리로 저렴한 Zukebox 만들기

https://www.obvoso.site/articles/youtube-api%EB%A1%9C-next.js-%EB%B8%94%EB%A1%9C%EA%B7%B8%EC%97%90-bgm-%EB%A7%8C%EB%93%A4%EA%B8%B0

**Comment by Feedbaek** [comment]
안녕하세요 신기하고 재미있는 기능 잘 봤습니다.

플레이리스트를 여러 개 미리 등록하고 사용자가 선택할 수 있도록 할 수도 있나요? 

감사합니다.

**Reply by obvoso** [reply]
안녕하세요 한달이 지나 댓글을 발견했습니다

환경변수에 PLAYLIST_ID를 저장하고 관리하고 있기 때문에, 
여러개의 플레이리스트를 만들어 변수로 관리한다면 런타임에 사용자의 선택에 따라 재생목록 정보를 갈아끼워주는 방식으로 구현이 가능할 거 같습니다.

제 블로그 노래가 취향에 안맞으셨나요? 나름 엄선한건데.... 
스물다섯 스물하나도 넣으면 취향에 맞으실까요? 

Feedbaek 감사합니다 고려해보겠습니다!
